docs/docs/external/getting-started/interop/asset-exchange.md

```diff
@@ -55,15 +57,134 @@ run the following steps:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=5 --recipient=alice --locker=bob --target-network=network2 --contract-id=<contract-id> --secret=<hash-pre-image>
        ```
-     - Run the following to verify the claim:
+     - Run the following to trigger `bob`'s claim for `bond01:a03` locked by `alice` in `network1`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=6 --recipient=bob --locker=alice --target-network=network1 --param=bond01:a03 --secret=<hash-pre-image>
        ```
-     - Run the following to trigger `bob`'s claim for `bond01:a03` locked by `alice` in `network1`:
+       
+     Above steps complete a successful asset exchange between two Fabric networks. 
+     In addition to above commands, following are the extra options:
+     - _In case_, `alice` wants to unlock the bond asset, run the following to trigger `alice`'s re-claim for `bond01:a03` locked in `network1`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=7 --locker=alice --recipient=bob --target-network=network1 --param=bond01:a03
        ```
-     - Run the following to verify the claim:
+     - _In case_, `bob` wants to unlock the token asset, run the following to trigger `bob`'s re-claim for `token1:100` locked in `network2`:
        ```bash
        ./bin/fabric-cli asset exchange-step --step=8 --locker=bob --recipient=alice --target-network=network2 --contract-id=<contract-id>
        ```
+       
+## Fabric with Corda
+
+We will demonstrate asset exchange of a bond in fabric `network1` with tokens on `Corda_Network`.
```

Comment:
Make this consistent with the beginning of the previous section. Replace with something like `A Fabric network transfers a bond from Alice to Bob in exchange for a transfer of tokens from Bob to Alice in a Corda network`.